### PR TITLE
DC - updated the main README for the site

### PR DESCRIPTION
**Pull request by aughera**
This is the first update to the documentation - main page only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 ## Description
 
-Firewall on Demand applies via NETCONF, flow rules to a network
-device. These rules are then propagated via e-bgp to peering routers.
-Each user is authenticated against shibboleth. Authorization is
-performed via a combination of a Shibboleth attribute and the peer
-network address range that the user originates from. FoD is meant to
-operate over this architecture:
+Firewall on Demand (hereafter FoD) is based on the [flowspy](https://github.com/grnet/flowspy) project developed by [GRNET](http://www.grnet.gr/).
```
Comment (aughera) [inline README.md]: This is just phrasing and a respectful call-out to both GRNET and flowspy.

File: README.md [hunk 2]
```diff
@@ -23,36 +23,35 @@ operate over this architecture:
                               +------------+
 
 
-NETCONF is chosen as the mgmt protocol to apply rules to a single
```
Comment (aughera) [inline README.md]: Not sure this is necessary

File: README.md [hunk 3]
```diff
@@ -23,36 +23,35 @@ operate over this architecture:
...
 ## Rest Api
-FoD provides a rest api. It uses token as authentication method. For usage
-instructions & examples check the documentation.
+FoD provides a rest api. It uses token as authentication method. For usage instructions & examples check the documentation.
 
 ## Limitations
 
 A user can belong to more than one `Peer` without any limitations.
-FoD UI polls the server to dynamically update the dashboard and the
+The FoD UI polls the server to dynamically update the dashboard and the
```
Comment (aughera) [inline README.md]: Should we say "the FoD server" instead?

Comment (dschmitz2017) [reply README.md]: Yes, good idea.

File: README.md (hunk 2 above)
Comment (dschmitz2017) [reply README.md]: As FoD currently does not support updating directly routers via BGP, 
it is maybe good to keep some notice that only NETCONF (Juniper-specific YANG model) is supported.
We may put a link to https://github.com/GEANT/FOD/blob/python3/doc/prerequisites/generic.md
for more information.

File: README.md (hunk 1 above)
Comment (dschmitz2017) [reply README.md]: Very good.

**Pull request merged**